### ISSUE TITLE
New parsec website incompatibility

### ISSUE THREAD

**Issue by mfouesneau** [mfouesneau/ezpadova#9]
5378a3e forces the old web service. We need to update the code to use the new version.

**Comment by JonasKemmer** [comment]
Seems like, there is now a mayor change in the retrieved text. No matter which web service is queried, I get an error:

```python
from ezpadova import parsec

parsec.get_one_isochrone(4e9, 0.004)
Interrogating http://stev.oapd.inaf.it...
Downloading data...http://stev.oapd.inaf.it/tmp/output690555869908.dat
Traceback (most recent call last):

  File "<ipython-input-17-552a3f5149a5>", line 1, in <module>
    parsec.get_one_isochrone(4e9, 0.004)
 
  File "/../ezpadova-1.0-py3.7.egg/ezpadova/parsec.py", line 352, in get_one_isochrone

  File "/.../ezpadova-1.0-py3.7.egg/ezpadova/parsec.py", line 276, in __convert_to_Table

  File "/.../ezpadova-1.0-py3.7.egg/ezpadova/parsec.py", line 272, in find_data

IndexError: string index out of range
```

**Comment by mfouesneau** [comment]
This is not a big in this package.

It seems that today their website returns empty files (only the header is returned).

The new version you of their service is not compatible (yet) but the older ones are still running.

Hopefully they will repair soon. I keep this issue open to keep ppl informed

**Comment by emilyhunt** [comment]
Hey, thanks for making this package! I think I might have found a way to update it to work with the latest website - it would take a rewrite and a different package to power the html stuff ([mechanize](https://mechanize.readthedocs.io/en/latest/index.html)) unfortunately, but should be possible. I got the idea [from this StackOverflow](https://stackoverflow.com/questions/8560959/using-python-to-sign-into-website-fill-in-a-form-then-sign-out) on automating filling of web forms in Python.

The `mechanize` library has a browser, which I think can access html forms that `urllib` can't and would solve the issues with the latest site. I tried:

```
import mechanize

browser = mechanize.Browser()
browser.open("http://stev.oapd.inaf.it/cgi-bin/cmd")

print(browser.forms()[0])
```

which output a list of all forms on the website, including some that aren't modifiable with the current version of `ezpadova` (like extinction, or output of different types like simulated populations), implying that it would be possible to fill them out with mechanize and automate download of the latest isochrones again.

Apologies if this is something you've already considered, I couldn't see it anywhere so figured I'd mention it just in case. I don't have time to work on this right now but would be happy to have a go in a few weeks' time and submit a PR, unless anyone else would rather have a go before then. Might also be worth thinking about how this would fit nicely into the module, I guess both the new and old query methods could be kept.
